kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -205,6 +222,10 @@ public String getSsea() {
     return getString(SSEA_CONFIG);
   }
 
+  public CannedAccessControlList getCannedAcl() {
+    return CannedAccessControlList.valueOf(CANNED_ACL_CONFIG);
```

Comment:
I need to come back to making a map that gets us from toString values to `name` values.